src/Particle/ParticleSetPool.cpp

```diff
@@ -32,7 +32,7 @@
 namespace qmcplusplus
 {
 ParticleSetPool::ParticleSetPool(Communicate* c, const char* aname)
-    : MPIObjectBase(c), SimulationCell(nullptr), TileMatrix(0)
+    : MPIObjectBase(c), TileMatrix(0)
```

Comment:
SimulationCell is no longer initialized with this constructor. Why?

Reply:
SimulationCell is unique_ptr and initialized as nullptr by default.